target/linux/ath79/config-5.4

```diff
@@ -212,6 +212,7 @@ CONFIG_REGMAP=y
 CONFIG_REGMAP_MMIO=y
 CONFIG_RESET_ATH79=y
 CONFIG_RESET_CONTROLLER=y
+# CONFIG_RTC_DRV_JZ4740 is not set
```

Comment:
Is that one needed for this device in particular?

Reply:
i'm not quite sure but every time i build a new image i got asked by the buildscript:
`Ingenic JZ4740 SoC (RTC_DRV_JZ4740) [N/m/y/?] (NEW)`

i would suspect that any other ath79-dev would have noticed this by now and i thought it might have something to do with the kmod-rtc-pcf8563 package

Reply:
Will have a look when I do my test-build.

Reply:
i have to mention though, i got that one when using j1 V=s for build error analysis.